### ISSUE TITLE
Too low contrast on some UI elements

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#7353]
### Steps to reproduce

How to check contrast with Firefox:  
2\. login at https://ocis.ocis-web.latest.owncloud.works  
3\. inspect site with Firefox Dev Tools > Tab "Accessibility" > check for issues "Contrast"  
4\. Select one of the found contrast violations  
5\. usually contrast should be chosen 4.5:1 (learn more: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast)

## How to reproduce / inspect in Firefox:

![Screenshot 000127@2x](https://user-images.githubusercontent.com/26610733/181206096-683769e7-baaa-4032-8f14-966ebae1a2b3.png)

## Access details

![Screenshot 000125@2x](https://user-images.githubusercontent.com/26610733/181202969-4de7a7e0-5456-43b2-928c-1db5453ad88b.png)

## Upload widget

![Screenshot 000122@2x](https://user-images.githubusercontent.com/26610733/181203733-e15cf16b-0750-4eeb-a77b-47ea218d59ba.png)

## Custom permissions

![Screenshot 000121@2x](https://user-images.githubusercontent.com/26610733/181203739-89176d86-d9d4-46b8-9127-a06c0098afd2.png)

## search active

![Screenshot 000119@2x](https://user-images.githubusercontent.com/26610733/181203741-65c1b28d-3849-4992-8e84-046a394c1bb0.png)

## search placeholder

![Screenshot 000120@2x](https://user-images.githubusercontent.com/26610733/181203743-f8c64690-33e9-4aea-9a25-3413ea7de7fb.png)

## search parent folder

![Screenshot 000118@2x](https://user-images.githubusercontent.com/26610733/181203746-231fed2a-c394-4c9c-b98e-4ffe24bfebb2.png)

## User Management

![Screenshot 000129@2x](https://user-images.githubusercontent.com/26610733/181231812-5cca88b7-ebe5-463c-bb1d-8bf983c3b9b3.png)

## Shared with others: parent folder

![Screenshot 000132@2x](https://user-images.githubusercontent.com/26610733/181234813-e2a4d3a9-3d6d-468a-9694-a238202106de.png)

![](https://user-images.githubusercontent.com/26610733/182148487-0bae1450-f84d-4268-bbe6-c948d67a6c9f.png)

### Expected behaviour

Contrasts should meet **WCAG 2.1 AA** criteria (https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast)

### Actual behaviour

Too low contrast on some UI elements
